docs/contributing.rst

```diff
@@ -9,12 +9,11 @@ First of all: thank you!
 Contributions can be made in different ways, not only code!
 As we follow the `all-contributors`_ specification, any contribution will be recognised accordingly.
 
-The first thing you might want to do, is to have a look at the `contributor guide <contributorfile.html>`_ page as well as the `code of conduct <conduct.html>`_.
+Follow these steps to get started:
 
-The second thing is to check you have ``git`` and ``pip`` installed in your system.
-
-The third thing is to install ``phys2bids`` as a developer.
-This will let you run the program with the latest modification, without requiring you to re-install it every time.
+1. Have a look at the `contributor guide <contributorfile.html>`_ page as well as the `code of conduct <conduct.html>`_.
+2. Make sure that you have a GitHub account. You can set up a `free GitHub account <https://github.com/>`_; here are some `instructions <https://help.github.com/articles/signing-up-for-a-new-github-account>`_.
+3. If you intend to contribute code and/or use the ``physiopy`` packages in any way, check that you have ``git`` and ``pip`` installed on your system. Then install ``phys2bids`` as a developer. This will let you run the program with the latest modification, without requiring to re-install it every time.
```

Comment:
```suggestion
3. If you intend to contribute code and/or use the ``physiopy`` packages in any way, check that you have ``git`` and ``pip`` installed on your system. Then install ``phys2bids`` as a developer. This will let you run the program with the latest modifications, without requiring to re-install it every time.
```